### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by Swizbiz**
Part of https://github.com/artipie/artipie/issues/1031

clean up readme

### CODE REVIEW

**Comment by Swizbiz** [comment]
@artipie/members please check PR

File: README.md [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
 [![Hits-of-Code](https://hitsofcode.com/github/artipie/http)](https://hitsofcode.com/view/github/artipie/http)
+[![Maven Central](https://img.shields.io/maven-central/v/com.artipie/http.svg)](https://maven-badges.herokuapp.com/maven-central/com.artipie/http)
+[![PDD status](http://www.0pdd.com/svg?name=artipie/http)](http://www.0pdd.com/p?name=artipie/http)
 
-Artipie HTTP base interfaces.
+# HTTP
+
+Artipie HTTP base interfaces  
+
+If you have any question or suggestions, do not hesitate to create as issue or contact us in
```
Comment (dgarus) [inline README.md:18]: typo: `as issue` -> `an issue`

File: README.md [hunk 2]
```diff
@@ -1,14 +1,22 @@
...
+[Telegram](https://t.me/artipie), our roadmap is available [here](https://github.com/orgs/artipie/projects/3).
```
Comment (dgarus) [inline README.md:19]: Maybe `our roadmap is` should be a separate sentence?

**Review by dgarus — CHANGES_REQUESTED** [review]
@Swizbiz, there are minor comments, please, have a look.

**Comment by Swizbiz** [comment]
@dgarus done, please check again

**Review by dgarus — APPROVED** [review]

**Comment by dgarus** [comment]
@Swizbiz thanks!
LGTM

@olenagerasimova, could you please check and merge?

File: README.md [hunk 3]
```diff
@@ -1,14 +1,23 @@
...
+[![License](https://img.shields.io/badge/license-MIT-green.svg)](https://github.com/artipie/http/blob/master/LICENSE.txt)
+[![codecov](https://codecov.io/gh/artipie/maven-adapter/branch/master/graph/badge.svg)](https://codecov.io/gh/artipie/maven-adapter)
 [![Hits-of-Code](https://hitsofcode.com/github/artipie/http)](https://hitsofcode.com/view/github/artipie/http)
+[![Maven Central](https://img.shields.io/maven-central/v/com.artipie/http.svg)](https://maven-badges.herokuapp.com/maven-central/com.artipie/http)
+[![PDD status](http://www.0pdd.com/svg?name=artipie/http)](http://www.0pdd.com/p?name=artipie/http)
 
-Artipie HTTP base interfaces.
+# HTTP
+
+Artipie HTTP base interfaces
```
Comment (dgarus) [inline README.md:16]: Dot is absent.

**Review by dgarus — CHANGES_REQUESTED** [review]
minor comment

**Pull request merged**